### DISCUSSION TITLE
Layout 里有个组件 侧边栏，英文名叫Sider，这个英文翻译可有不妥之处？

### DISCUSSION THREAD

**Discussion by shifenhutu** [ant-design/ant-design#39502] [General]
似乎不存在这个 Sider 单词  ， ide老是提醒  Typo error

请问 侧边栏翻译为 sider 可有不妥之处？   可有更好的单词来代替 ？

我查询了一下，发现似乎大家的用词都是sider，莫非是个历史遗留的问题 ？ 

谢谢

我英文不太好 ~~~

**Comment by BoyYangzai** [comment]
`side` 有名词`侧，侧面，边`的意思，动词也是`侧`，`sidebar` `侧边栏`也比较常用，好像确实没有Sider这个单词，但是我觉得布局组件里用 `Sider` 来表示侧边的布局确实挺形象的 ovo
